LanguageServer/Protocol/Document/Server/IFoldingRangeHandler.cs

```diff
@@ -0,0 +1,11 @@
+using OmniSharp.Extensions.JsonRpc;
+using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+// ReSharper disable once CheckNamespace
+namespace OmniSharp.Extensions.LanguageServer.Protocol.Server
+{
+    using static DocumentNames;
+    [Serial, Method(FoldingRange)]
```

Comment:
Is `Serial` required?  Generally for "answering questions" `Parallel` behaves nicer because we can return results back to the editor faster.

Reply:
Sorry for the late answer. While implementing these classes, I've had a look at other classes and tried to follow what they do..